### ISSUE TITLE
I hope to add viwer accounts

### ISSUE THREAD

**Issue by addname** [openobserve/openobserve#2310]
### Which OpenObserve functionalities are relevant/related to the feature request?

login, home, log search, dashboards, users, streams, functions, ingestion, alerts, localization, routing, metrics, traces, other

### Description

I  hope to add viewer accounts that can only view various types of data but cannot modify the data. This is to prevent inadvertent actions, deletions, or modifications of data.

### Proposed solution

1

### Alternatives considered

1

**Comment by gaby** [comment]
@prabhatsharma This is similar to what I mentioned about having a "read-only" role.

**Comment by addname** [comment]
> @prabhatsharma This is similar to what I mentioned about having a "read-only" role.
Yes, there is an administrator role that can manage all data, and there is another viewer role that is limited to 'read-only.' Such a simple design can solve 80% of the permission issues.

**Comment by gaby** [comment]
Related: https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/#basic-roles

**Comment by Mart-Bogdan** [comment]
Documentation https://openobserve.ai/docs/user-guide/users/#create-user says  that roles are admin and member, but I can only see admin when downloaded docker `latest` is that a bug or intended now?

**Comment by pk0ps** [comment]
> Documentation https://openobserve.ai/docs/user-guide/users/#create-user says that roles are admin and member, but I can only see admin when downloaded docker `latest` is that a bug or intended now?

Same problem: there is no `member` role in the **Add user** form.

Image:
```
public.ecr.aws/zinclabs/openobserve:v0.10.7
sha256:e00842be3c6793cf3194a0b95c481f979717a32c9471acb163595bcf9e74aeca
```

![Screenshot 2024-07-15 at 19 13 40](https://github.com/user-attachments/assets/2b115b1b-c6cd-40cb-8b73-97b378877126)

**Comment by DrBlury** [comment]
Push, still an issue. Would love to help but sadly no rust knowledge here.

**Comment by hengfeiyang** [comment]
We only have `admin` for open source, if you want more control of RBAC, you can try enterprise version.

**Comment by gaby** [comment]
@hengfeiyang An enterprise docker-compose or more simple standalone helm chart would be useful.

The current chart that has enterprise runs a lot of containers and is meant for clusters.

**Comment by hengfeiyang** [comment]
@gaby Maybe you can create an issue at here: https://github.com/openobserve/openobserve-helm-chart/
